sdk/monitor/Azure.Monitor.Query/src/LogsQueryOptions.cs

```diff
@@ -2,12 +2,25 @@
 // Licensed under the MIT License.
 
 using System;
+using Azure.Monitor.Query.Models;
 
 namespace Azure.Monitor.Query
 {
+    /// <summary>
+    /// Options for <see cref="LogsClient.QueryAsync"/> that allow specifying the service timeout or whether to include
+    /// the query execution statistics.
+    /// </summary>
     public class LogsQueryOptions
     {
+        /// <summary>
+        /// Gets or sets the value indicating the service timeout for the query. Defaults to <c>null</c>.
```

Comment:
Should we say what the default is rather than the default value of the type? For example, `Defaults to the HttpClient default ... `

Reply:
That's an interesting question.  I was trying to avoid presuming what service does as it might change with the service version.

Reply:
How would someone know what to set this to?

Reply:
Their query would fail. No matter whether we give them a default value or not they need to see their query timing out to start setting this value.